site/app/templates/course/CourseMaterials.twig

```diff
@@ -9,13 +9,13 @@
     {% if user_group == 1 %}
         <div style="float: right; margin-bottom: 20px;">
             {# button to open all of the subfolders and cookie it#}
-            <a onclick='setCookie("foldersOpen",openAllDivForCourseMaterials());' class="btn btn-primary key_to_click" tabindex="0">Open/Close All Folders</a>
+            <button onclick='setCookie("foldersOpen",openAllDivForCourseMaterials());' class="btn btn-primary key_to_click" tabindex="0">Open/Close All Folders</button>
```

Comment:
I'm seeing some visual changes in the course materials pages I think this. I think there are some issues where some of the buttons don't have any text within the button so maybe some of the icons need to be refactored so that they are nested within the button instead of containing the anchor tag

<img width="1177" alt="image" src="https://user-images.githubusercontent.com/12129065/170621354-3ad6488e-9c83-447e-bb95-9963dcc5fbcf.png">